apps/communications/dialer/test/marionette/lib/views/activities/views.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/* global module */
+
+var ActivitiesAccessor = require('./accessors');
+
+//and list the rest of accessors needes, like the other POs to go I guess
+
+//var ComposerAccessor = require('../shared/composer_accessors');
+//var MenuAccessor = require('../shared/menu_accessors');
+//var Tools = require('../shared/tools');
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var fromApp = require(appRoot +
```

Comment:
Oh, this include is not needed anymore -- I discovered in back in Orlando. There is a simpler way to import the `Contacts` page class:
```js
contactsApp = this.client.loader.getAppClass('contact')
```